docker/root/etc/s6/gitea/setup

```diff
@@ -41,7 +41,7 @@ if [ ! -f ${GITEA_CUSTOM}/conf/app.ini ]; then
     DISABLE_REGISTRATION=${DISABLE_REGISTRATION:-"false"} \
     REQUIRE_SIGNIN_VIEW=${REQUIRE_SIGNIN_VIEW:-"false"} \
     SECRET_KEY=${SECRET_KEY:-""} \
-    envsubst < /etc/templates/app.ini > ${GITEA_CUSTOM}/conf/app.ini
+    envsubst < /etc/templates/gitea.ini > ${GITEA_CUSTOM}/conf/app.ini
```

Comment:
There is no `gitea.ini` in `docker/root/etc/templates/`

Reply:
you need to check the files changed and you'll find it

Reply:
here see: docker/root/etc/templates/app.ini → docker/root/etc/templates/gitea.ini

Reply:
Sorry I missed that.

I think that's a breaking change unfortunately - it's already indirected from /etc/templates/app.ini to ${GITEA_CUSTOM}/conf/app.ini so I'm not certain that it needs to be changed.

Reply:
Why is it a breaking change? I can't see anything broken in the container that I built using this and I have online right now. What am I missing?